buildchain/buildchain/docker_command.py

```diff
@@ -77,6 +77,24 @@ def decorated_task(*args: Any, **kwargs: Any) -> Optional[TaskError]:
     return wrapped_task
 
 
+# The call method is not counted as a public method
+# pylint: disable=too-few-public-methods
```

Comment:
Do you still need this one? Seems not required for `DockerRmi` which has the same number of method.

Reply:
Stray rebase removal, it's needed on both (at least it is locally).

Reply:
You sure? `doit.sh lint` doesn't complain, even locally for me.
Are you sure it's not your weird editor linter that complains :p ?

Since we froze the version of `pylint` we should have the same results.

Reply:
Ok, seems like our `disable` are not scope to the class, but global to the file.

Either we got the syntax wrong, either it's a bug in pylint. Will need to check the doc…

Reply:
Ok, the correct syntax seems to be:

```
class DockerExists:
    # The call method is not counted as a public method
    # pylint:disable=too-few-public-methods
```

To disable it only for the class and not the whole file (I also noticed that we usually don't put spaces before `disable`, but that's minor).

Reply:
Removed the need for these by switching to functions.